packages/ember-routing-htmlbars/lib/keywords/link-to.js

```diff
@@ -315,19 +313,13 @@ export default {
   },
 
   render(morph, env, scope, params, hash, template, inverse, visitor) {
-    var attrs = merge({}, hash);
-
-    // TODO: Rewrite link-to to use arbitrary length positional params.
-    attrs.params = readArray(params);
-
-    // Used for deprecations (to tell the user what view the deprecated syntax
-    // was used in).
-    attrs.view = env.view;
-
-    // TODO: Remove once `hasBlock` is working again
-    attrs.hasBlock = !!template;
+    Ember.runInDebug(() => {
```

Comment:
I believe the import would still exist, but I think that is likely fine.  Using a symbol here seems good (and helps prevent folks that we are going to start encouraging to use `Ember.LinkComponent.extend` from relying on the private internals...).